authz/authz.proto

```diff
@@ -230,6 +228,16 @@ message ProbeRequest {
   // It has to be a fully qualified name, like:
   //    "/gnsi.ssh.Ssh/MutateHostCredentials"
   string rpc = 2;
+
+  // The profile for which the authz policy is being probed. In the case that
+  // this field is not specified, the default authz policy which applies to all
+  // gRPC endpoints run by the target is assumed. Where non-default policies
+  // are to supported by an endpoint, the value of the profile determines which
+  // set of policies are to be probed.
+  //
+  // Note that the authz profile is considered independent from a SSL profile
+  // ID (as referenced by gnsi.Certz).
+  string authz_profile_id = 3;
```

Comment:
I discussed with Marcus too. I think this makes sense to have here so that you can know exactly what authz policy you're probing.

However, can we add a clarification that says that you can only have one rotate operation in progress at once, to avoid the complexity of the state machine having to handle "what happens if you start multiple operations and then probe another one".

Reply:
I made a similar change for Certz here: https://github.com/openconfig/gnsi/pull/201